### DISCUSSION TITLE
TSC Meeting November 2024

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#9637] [Announcements]
Now that SOF v2.11 has been released its time for the @thesofproject/steering-committee to meet and align direction for v2.12 and future releases.

Please let us know your availability here: https://doodle.com/meeting/participate/id/bkDqvwNd

**Tentative Agenda** (TSC members can propose items and add more context to existing items).

1. Member updates.

2. Release cadence and versioning to better align with Zephyr release cadence and features.

3. Zephyr integration plans and next steps.

4. High level firmware feature road map for next 2 releases. Alignment and discussion of upcoming FW features for infrastructure, audio processing, SIMD, deprecation, legacy platform/feature removals.

5. High level driver road map for next 2 releases. Alignment and discussion of upcoming driver work, cleanups, new feature support for llext modules.

6. High level tooling road map for next 2 releases. Alignment on tooling, matlab/octave, testbench.

7. SDCA library driver integration. Alignment to build SDCA support into general kernel, FW and infra. 

8. Zephyr Multimedia Framework

**Comment by dbaluta** [comment]
@lgirdwood  please use this link https://doodle.com/meeting/participate/id/dyMLpDWd

Doodle allows us only 20 slots, but I think it should be fine.

**Reply by cujomalainey** [reply]
I am in Taiwan next week. So those times won't work well for me.

**Reply by dbaluta** [reply]
> I am in Taiwan next week. So those times won't work well for me.

I can also add a new doodle for week 18 - 22 if necessary. Will have to manage two separate doodles as the free version is restricted to only 20 slots.

**Reply by lgirdwood** [reply]
@dbaluta This week is probably too early, next week is out due to travel so lets schedule sometime 18th - 22nd. Can you also put a time slot for 2pm - 3pm UK time as this is an overlap time for Asia and West Coast US. Thanks !

**Reply by dbaluta** [reply]
@lgirdwood  Let's use this doodle : https://doodle.com/meeting/participate/id/bkDqvwNd

**Comment by dbaluta** [comment]
@lgirdwood  can you please help send an email to interested parties?

**Reply by lgirdwood** [reply]
Will do - I think folks back from travels now.

**Comment by dbaluta** [comment]
@lgirdwood  so I think best time for anyone is tomorrow,Thursday 18:00 EET. That is 16:00 UK time.

Can you please send a meeting invite?

**Reply by lgirdwood** [reply]
> @lgirdwood so I think best time for anyone is tomorrow,Thursday 18:00 EET. That is 16:00 UK time.
> 
> Can you please send a meeting invite?

@dbaluta I don't think we have quorum, I know we have had members on travel and Thanksgiving is 28th Nov. We should move for the week after Thanksgiving in the US (e.g. 2nd - 6th Dec). It will be easier to get Quorum. :)

@thesofproject/steering-committee lets push back two weeks, last few weeks have been difficult to schedule a call with Thanksgiving and holidays. It looks like Thursday may be the best day for schedule given the doodle data.

**Reply by dbaluta** [reply]
@lgirdwood  Agree.

**Reply by cujomalainey** [reply]
Unfortunately that will fall past my transition date. For those who haven't heard yet I will be transitioning to another team in Google and @johnylin76  will be taking over all my duties, I don't think I will be able to make it to the meeting as I will be doing a lot of training and ramp up.

https://github.com/thesofproject/sof-docs/pull/504

It has been wonderful working with all of you and I am very proud of the project we have built. I will be doing more work in the zephyr tree going forward so we may cross paths again. I wish you all the best of luck.

**Reply by lgirdwood** [reply]
Thank you @cujomalainey, we have absolutely built a better project with your support.
